store/modules/events.ts

```diff
@@ -0,0 +1,77 @@
+type Event = {
```

Comment:
Non-blocking: in the context of Web applications, I tend to mistake `Event` with a JS event. Would it be possible to rename it to `CommunityEvent` or something similar?